demo-app/ng8/src/app/pages/reactive-forms-example/reactive-forms-example.component.ts

```diff
@@ -0,0 +1,92 @@
+/*tslint:disable:template-use-track-by-function template-cyclomatic-complexity*/
+import { Component } from "@angular/core";
+import { AbstractControl, FormBuilder, FormControl, FormGroup, ValidationErrors, Validators } from "@angular/forms";
+import { ErrorStateMatcher } from "@angular/material/core";
+import { PasswordValidator } from "../../password-validator";
+import { ParentErrorStateMatcher } from "../../parent-error-state-matcher";
+
+@Component({
+	selector: "app-reactive-forms-example",
+	templateUrl: "./reactive-forms-example.component.html",
+	styleUrls: ["./reactive-forms-example.component.scss"]
+})
+export class ReactiveFormsExampleComponent {
+	public formGroup: FormGroup;
+	public validationMessages: { [key: string]: { type: string; message: string }[] };
+	public parentErrorStateMatcher: ErrorStateMatcher = new ParentErrorStateMatcher();
+	public passwordPattern = "^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])[a-zA-Z0-9]+$";
+	public showValidationDetails = false;
+	public showValidationSummary = true;
+
+	public constructor(private formBuilder: FormBuilder) {
+		this.formGroup = this.formBuilder.group({
+			username: [undefined, Validators.required],
+			matchingPasswords: new FormGroup(
+				{
```

Comment:
The indentation is not the same in both demo-apps (ng7, ng8).
Could you please fix it and to have "2 spaces" everywhere ? 😊

Reply:
Sorry, I don't get it. What "2 spaces"?

I've ran Prettier on the whole project and the 2 demo-apps have exactly the code formatting...

Reply:
@christophercr my apologies...
I did the review with GitHub and I thought that the indentation was incorrect. But it's not. So please forget this comment 😊